src/pages/Main/MainPage.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import MainSlide from './MainSlide';
+import Product from './weeklybest/Product';
+import MAIN_SLIDE from './mainSlideData';
+import './MainPage.scss';
+
+class MainPage extends React.Component {
+  state = {
+    slide: [],
+    Products: [],
+    bestIndex: 0,
+    slideIndex: 0,
+  };
+
+  prevSlide = () => {
+    if (this.state.slideIndex > 0) {
+      this.setState({
+        slideIndex: this.state.slideIndex - 1,
+      });
+    } else {
+      this.setState({ slideIndex: 3 });
+    }
+  };
+
+  nextSlide = () => {
+    if (this.state.slideIndex < 3) {
+      this.setState({
+        slideIndex: this.state.slideIndex + 1,
+      });
+    } else {
+      this.setState({ slideIndex: 0 });
+    }
+  };
+
+  prevButton = () => {
+    if (this.state.bestIndex === 0) return;
+    this.setState({
+      bestIndex: this.state.bestIndex - 1,
+    });
+  };
+
+  nextButton = () => {
+    if (this.state.bestIndex === 1) return;
+    this.setState({
+      bestIndex: this.state.bestIndex + 1,
+    });
+  };
+  componentDidMount() {
+    fetch('http://localhost:3000/data/weeklyBest.json')
```

Comment:
```suggestion
    fetch('/data/weeklyBest.json')
```
localhost:3000 필요없습니다!

Reply:
데이터 연결하고 수정하였습니다!

Reply:
데이터 연결하면서 수정했습니다!